facility-manager.html

```diff
@@ -0,0 +1,806 @@
+
+<p>Click <span class="bold">Save</span> to complete the change.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerRemoveUser.jpg" class="align-center" src="images/facilityManagerRemoveUser.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+</ol>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h2>Moving Data</h2>
+
+<p>When a user is removed from a group, any data they have in the group remains in the group, and is still owned by the user.</p>
+
+<p>Data can be moved to another group by the user before they are removed, or by the group owner or administrator at any time.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.web</h3>
+
+<ol>
+
+<li>
+
+<p>Select <span class="bold">All Members</span> from the group display menu to view data owned by a user who is no longer a member of the group.</p>
+
+<p>Right-click on the data to be moved and select <span class="bold">Move to Group ... </span>.</p>
+
+<p>Select the destination from the pop-up window.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerMoveWeb.jpg" class="align-center" src="images/facilityManagerMoveWeb.jpg" style="width: 100%;" />
+
+</li>
+
+</ol>
+
+<p class="top_link"><a href="#top"><img src="images/back_to_top.png" alt="Image for link to top of page" style="float: right;"/></a></p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<h3>OMERO.insight</h3>
+
+<ol>
+
+<li>
+
+<p>Uncheck <span class="bold">Display Users</span> from the Display Groups menu to view data owned by a user who is no longer a member of the group.</p>
+
+<p>Right-click on the data to be moved and select <span class="bold">Move to Group ... </span>.</p>
+
+<p>Select the destination group from options.</p>
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+<img alt="../images/facilityManagerMoveInsight.jpg" class="align-center" src="images/facilityManagerMoveInsight.jpg" style="width: 100%;" />
+
+<div class="line-block">
+<div class="line"><br /></div> <!-- end #line -->
+</div> <!-- end #line-block -->
+
+</li>
+
+</ol>
+
+<div class="note">
+<p class="note-title">Note</p>
+<p class="note-text">Using the Command Line Interface (CLI), administrators and group owners (for data in their groups)  can:</p>
+
+<ul>
+
+<li>view and move data owned by users no longer a member of a group</li>
```

Comment:
Viewing data on the CLI seems a bit odd because elsewhere you use view to mean visualise. I'd be inclined to just say move, or use 'list' and move as in https://www.openmicroscopy.org/site/support/omero5.2/sysadmins/cli/fs.html

You might also want to reference https://www.openmicroscopy.org/site/support/omero5.2/users/cli/chgrp.html here as the CLI guides can be a little hard to find